### ISSUE TITLE
Add assertDatabaseMissingEach helper to the InteractsWithDatabase trait.

### ISSUE THREAD

**Issue by Denis4yk** [laravel/internals#948]
This could be quite useful.

Sometimes I want to be sure, that **not a single** field is present in the DB.

This

```php
$this->assertDatabaseMissing('menu_items', [
    'menu_id' => $fakeUuid,
    'sort_order'   => 2,
]);
```

will give an error, when both fields are performed. And if I have only a `fakeUuid`, I won't know about it.

**Comment by deleugpn** [comment]
I would expect that a method called `assertDatabaseMissingEach` would take a collection of array of attributes, where each row is a potential record and call `assertDatabaseMissing` for each row.

**Comment by Denis4yk** [comment]
@deleugpn 

Yes, that's what I am talking about.

**Comment by deleugpn** [comment]
No, you're talking about going through each field whereas I'm talking about going through each *set* of fields.

$this->assertDatabaseMissingEach('menu_items', [
    ['menu_id' => $fakeUuid,    'sort_order'   => 2],
    ['menu_id' => $fakeUuid,    'sort_order'   => 3],
    ['menu_id' => $fakeUuid,    'sort_order'   => 1],
    ['menu_id' => $anotherFakeUuid,    'sort_order'   => 5],
]);

And then `assertDatabaseMissingEach` would just call `assertDatabaseMissing` as is.

**Comment by Denis4yk** [comment]
I see. Id like to call `assertDatabaseMissing` on each _attribute_, but `assertDatabaseMissingEachAttribute` sounds too verbose)

Do you have a better naming idea?)

**Comment by Denis4yk** [comment]
I see. Id like to call `assertDatabaseMissing` on each _attribute_, but `assertDatabaseMissingEachAttribute` sounds too verbose)

Do you have a better naming idea?)

**Comment by Denis4yk** [comment]
I see. Id like to call `assertDatabaseMissing` on each _attribute_, but `assertDatabaseMissingEachAttribute` sounds too verbose)

Do you have a better naming idea?)
